canonical/snapcraft.io
# fix: Refactor models service to use account ID

## Done
Refactored the brand store so that the model service uses the brand account ID to call the API rather than the store ID.

## How to QA
- Go to https://snapcraft-io-4783.demos.haus/admin/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/models
- Check that the models have loaded
- Go to https://snapcraft-io-4783.demos.haus/admin/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/models/alimot_test_model_a
- Check that the model has loaded
- Go to https://snapcraft-io-4783.demos.haus/admin/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/models/alimot_test_model_a/policies
- Check that the polices have loaded
- Go to https://snapcraft-io-4783.demos.haus/admin/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/signing-keys
- Check that the signing keys have loaded
- Go to any of your stores and check that they look and behave as expected

## Testing

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-13713
